vscode-car-plugin/src/main/java/org/wso2/maven/CAppHandler.java

```diff
@@ -0,0 +1,243 @@
+/*
+Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+*
+* WSO2 Inc. licenses this file to you under the Apache License,
+* Version 2.0 (the "License"); you may not use this file except
+* in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+package org.wso2.maven;
+
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.List;
+import javax.xml.stream.XMLStreamException;
+
+import org.apache.axiom.om.OMDocument;
+import org.apache.axiom.om.OMElement;
+import org.apache.commons.io.FileUtils;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.project.MavenProject;
+import org.wso2.maven.Model.Artifact;
+import org.wso2.maven.Model.ArtifactDependency;
+import org.wso2.maven.core.model.AbstractXMLDoc;
+
+class CAppHandler extends AbstractXMLDoc {
+
+    /*
+     * Read /synapse-config/artifact.xml file and create corresponding files in archive directory.
```

Comment:
Adhere to javadoc convention https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-141999.html

Reply:
Fixed.